regression-tests.nobackend/rectify-axfr/command

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+MAKE=${MAKE:-make}
+source ../regression-tests/common
+
+# run PowerDNS using random backend
+
+rm -f pdns*.pid pdns-gsqlite3.conf pdns.sqlite3 rectify-axfr/zones/* named.conf
+cp -f ../regression-tests/zones/* rectify-axfr/zones/
+
+sed '/directory/ { s@./zones@./rectify-axfr/zones@ }' ../regression-tests/named.conf > ./named.conf
+
+cat > pdns-gsqlite3.conf << __EOF__
+launch=gsqlite3
+gsqlite3-database=pdns.sqlite3
+gsqlite3-dnssec
+module-dir=../regression-tests/modules
+__EOF__
+
+ARGS="--config-dir=. --config-name=gsqlite3"
+
+port=5501
+
+sqlite3 pdns.sqlite3 < ../modules/gsqlite3backend/schema.sqlite3.sql
+tosql gsqlite | sqlite3 pdns.sqlite3
+echo ANALYZE\; | sqlite3 pdns.sqlite3
+
+securezone ()
+{
+	local zone=$1
+	$PDNSUTIL $ARGS import-zone-key $zone $zone.private ksk 2>&1
+	$PDNSUTIL $ARGS add-zone-key $zone rsasha256 1024 zsk active 2>&1
+	$PDNSUTIL $ARGS set-publish-cds $zone 2>&1
+	$PDNSUTIL $ARGS set-publish-cdnskey $zone 2>&1
+	if [ "${zone: 0:20}" = "cdnskey-cds-test.com" ]; then
+		$PDNSUTIL $ARGS set-publish-cds $zone 2>&1
+		$PDNSUTIL $ARGS set-publish-cdnskey $zone 2>&1
+	fi
+}
+
+for zone in $(grep 'zone ' named.conf | cut -f2 -d\")
+do
+	$PDNSUTIL $ARGS set-nsec3 $zone "1 1 1 abcd" >&2
+	securezone $zone >&2
+	if [ "$zone" = "tsig.com" ]; then
+		$PDNSUTIL $ARGS import-tsig-key test $ALGORITHM $KEY >&2
+		$PDNSUTIL $ARGS activate-tsig-key tsig.com test master >&2
+	fi
+done
+
+$RUNWRAPPER $PDNS --daemon=no --local-port=$port --socket-dir=./          \
+        --no-shuffle $ARGS \
+        --allow-axfr-ips=127.0.0.1 \
+        --cache-ttl=60 --module-dir=../regression-tests/modules >&2 &
+
+check_process
+
+set +e
+
+for zone in $(grep 'zone ' named.conf | cut -f2 -d\" | grep -v '^\(example.com\|nztest.com\|tsig.com\)$')
```

Comment:
Is this block an exact copy of regression-tests/tests/verify-dnssec-zone/command ? If so, please use the existing command file here to avoid code duplication.